modules/health_quest/app/docs/health_quest/v0/health_quest.yaml

```diff
@@ -0,0 +1,251 @@
+openapi: '3.0.0'
+info:
+  version: 0.0.1
+  title: Health Quest
+  description: |
+    ## The API for the Healthcare Clipboard 
+
+    These endpoints include the Healthcare Experience Questionaire 
+    Responses and clipboard related endpoints into the MAP PGD and scehduled appointments
+
+  contact:
+    name: va.gov
+tags:
+  - name: appointments
+    description: Appointments
+  - name: systems
+    description: Systems
+servers:
+  - url: https://veteran.mobile.va.gov/smart-pgd-fhir/v1/
+    description: MAP API sandbox environment
+    variables:
+      version:
+        default: v0
+  - url: https://api.va.gov/health_quest/{version}
+    description: VA.gov API production environment
+    variables:
+      version:
+        default: v0
+paths:
+  '/appointments':
+    get:
+      tags:
+        - appointments
+      summary: Returns alist of online scheduling appointments
+      operationId: getAppointments
+      security:
+        - bearerAuth: []
+      responses:
+        '200':
+          description: Systems retrieved successfully
```

Comment:
What systems were retrieved, and why are we retrieving systems, and not `Appointments`, with the `getAppointments` operation?

Reply:
This was wrongfully set to Systems. I've set it to Appointments to correct the error.